test/test_gpu3.cpp

```diff
@@ -8099,6 +8099,69 @@ TEST_F(NVFuserTest, FusionManagedData_CUDA) {
   ASSERT_EQ(kernel->getManaged<T2>("data2").magic_number, 0x123456789abcdef);
 }
 
+TEST_F(NVFuserTest, FusionGeneratedTestCrossEntropyLoss_CUDA) {
+  std::unique_ptr<Fusion> fusion_ptr = std::make_unique<Fusion>();
+  auto fusion = fusion_ptr.get();
+  FusionGuard fg(fusion);
+
+  {
+    auto tv0 = TensorViewBuilder().ndims(2).contiguity({true, true}).dtype(DataType::Float).build();
+    fusion->addInput(tv0);
+    auto tv1 = TensorViewBuilder().ndims(1).contiguity({true}).dtype(DataType::Int).build();
+    fusion->addInput(tv1);
+    auto tv2 = max(tv0, {1});
+    auto tv3 = broadcast(tv2, {false, true});
+    auto tv4 = expand(tv3, 
+                      {IrBuilder::create<Int>(128),
+                       IrBuilder::create<Int>(371)});
+    auto tv5 = sub(tv0, tv4);
+    auto tv6 = exp(tv5);
+    auto tv7 = sum(tv6, {1});
+    auto tv8 = broadcast(tv7, {false, true});
+    auto tv9 = expand(tv8, 
+                      {IrBuilder::create<Int>(128),
+                       IrBuilder::create<Int>(371)});
+    auto tv10 = div(tv6, tv9);
+    auto tv11 = log(tv10);
+    auto tv12 = neg(tv11);
+    auto tv13 = reshape(tv1, {128}, {128, 1});
```

Comment:
nit: we can use `unsqueeze`

Reply:
yeah, the only reason we are seeing a reshape here is that unsqueeze/squeeze is not in thunder lowering yet.